<?xml version="1.0"?>
<robot xmlns:xacro="http://www.ros.org/wiki/xacro">

    <xacro:property name="chassis_radius"   value="0.1778"/>   <!-- 7 in = 0.1778 m -->
    <xacro:property name="chassis_thick"    value="0.05"/>     <!-- chassis height -->
    <xacro:property name="wheel_radius"     value="0.035"/>    <!-- ~70 mm dia. 0.035 -->
    <xacro:property name="wheel_width"      value="0.01"/>
    <xacro:property name="wheel_y_offset"   value="0.16"/>     <!-- offset from center -->
    <xacro:property name="caster_radius"    value="0.02"/> <!--0.02 -->
    <xacro:property name="caster_x_offset"  value="0.15"/>
    <xacro:property name="wheel_z" value="${(wheel_radius - (chassis_thick/2))-0.01}"/>


</robot>